src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ExceptionHandler.kt

```diff
@@ -78,6 +80,12 @@ class SendLegalMailToPrisonsApiExceptionHandler {
       .body(ErrorResponse(status = BAD_REQUEST, errorCode = MalformedRequest))
   }
 
+  @ExceptionHandler(DuplicateContactException::class)
+  @ResponseStatus(CONFLICT)
+  fun handleDuplicateContactException(e: DuplicateContactException) {
+    log.info { "Duplicate Contact exception: [${e.userId}, ${e.prisonNumber}]" }
```

Comment:
Hmmm, actually this is not quite right. The swagger spec says it will return an `ErrorResponse` body (because I blindly copy and pasted without really thinking about it !!). but this returns no body and just the status code.
Either need to change this implementation and create a suitable instance of `ErrorResponse`, or change the swagger spec....